package.json

```diff
@@ -14,7 +14,6 @@
     "start": "parcel src/index.html",
     "dev": "parcel --log-level verbose src/index.html",
     "generate": "npm-run-all -s generate:*",
-    "generate:pgs-be": "shx rm -rf src/generated/pgs && shx mkdir -p src/generated/pgs && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/c39cea0b3ac4d992f4bda0026365b5a0eac96802/src/core/api/payment_transactions_gateway_api/external/v1/_openapi.json.tpl --no-strict --out-dir src/generated/pgs",
```

Comment:
Are you sure you want to eliminate this autogeneration ?

Reply:
Hi Fernando,
this autogeneration was completely overwritten by the follower generate (you cannot see in this code portion) so it is completely unuseful. Anyway, we need to update infra json openapi spec and than we can refers it here (coming soon)